pkgs/development/libraries/isl/0.23.0.nix

```diff
@@ -0,0 +1,11 @@
+import ./generic.nix rec {
+  version = "0.23";
+  urls = [
+    "mirror://sourceforge/libisl/isl-${version}.tar.xz"
+    "https://libisl.sourceforge.io/isl-${version}.tar.xz"
+  ];
+  sha256 = "";
```

Comment:
What about using a real hash here?

Reply:
Wow.... I must have forgotten to add the changes, fixed sorry :)

Reply:
It happens in the best package managers.